lib/applications/validation.ts

```diff
@@ -215,13 +216,16 @@ export const reasonForReplacementFormSchema = object({
     .default(null)
     .when('reason', {
       is: 'LOST',
-      then: date()
-        .transform((_value, originalValue) => {
-          return new Date(originalValue);
-        })
-        .typeError('Please enter date APP was lost')
-        .max(new Date(), 'Date must be in the past')
-        .required('Please enter date APP was lost'),
+      then: lazy(() =>
```

Comment:
Addition of `lazy` is not directly related to this bug, preemptively adding it in case something like #328 comes up